### DISCUSSION TITLE
Using own database

### DISCUSSION THREAD

**Discussion by BeeTwenty** [immich-app/immich#7137] [Q&A]
### The bug

Im struggeling to merge database with my own where can I find the vectors extension?

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

LATEST

### Version of Immich Mobile App

LATEST

### Platform with the issue

- [ ] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
s
```

### Your .env content

```Shell
s
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by schuhbacca** [comment]
See here: https://github.com/immich-app/immich/discussions/6792

Also please don't submit a bug for a user issue. Instead try posting in the discord help section or discussion area of github.
